.dockerignore

```diff
@@ -28,6 +28,13 @@ node_modules
 build/*
 !build/libs/*.war
 
+#########################
+# exclude Artemis configs
+#########################
+/src/main/resources/config/application-local*.yml
+/src/main/resources/id_*
+/src/main/resources/known_hosts
```

Comment:
Is the beginning slash correct?

Reply:
It should be fine [according to the docs and it's examples](https://docs.docker.com/engine/reference/builder/#dockerignore-file).